### DISCUSSION TITLE
好消息，我们已经修复了 7.8.2 中的崩溃问题，版本 7.8.3 现已推出 https://github.com/getlantern/lantern   (Logs also here)

### DISCUSSION THREAD

**Discussion by MaxMKMax** [getlantern/lantern-forum-cn#300] [Announcements]
https://github.com/getlantern/lantern

![image](https://github.com/getlantern/lantern-forum-cn/assets/155475142/4d55f387-520a-491a-a6dd-f2802bc5905d)

**Comment by Jumpsteelrail** [comment]
我用的系统是WIN 10 系统，下载了这个链接下的最新版本，蓝灯图标的颜色依然是灰色闪退

**Reply by oo7lax1** [reply]
不知道几天能修复，我从上周末就是这样了

**Comment by casperwzy** [comment]
win10 系统，最新的7.8.3版本 ，安装后，还是闪退，请问在闪退的情况下，如何提交应用内报告， 然后给你 8 位数票号，我都打不开，双击进入，右下角刚出现图标就闪退了，另外 有没有闪退，使用新版本，之后正常的伙伴呢？给说一下是如何操作的吧，多谢了！

**Reply by carpediemcc** [reply]
我也是一样

**Reply by Robertzhuge** [reply]
一样的，根本看不到界面，我2个付费账号，快崩溃了

**Comment by casperwzy** [comment]
> https://github.com/getlantern/lantern
> 
> ![image](https://private-user-images.githubusercontent.com/155475142/318117723-4d55f387-520a-491a-a6dd-f2802bc5905d.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTE3NTY2NjUsIm5iZiI6MTcxMTc1NjM2NSwicGF0aCI6Ii8xNTU0NzUxNDIvMzE4MTE3NzIzLTRkNTVmMzg3LTUyMGEtNDkxYS1hNmRkLWYyODAyYmM1OTA1ZC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwMzI5JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDMyOVQyMzUyNDVaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1hZWI4NzA1Mzk5NDJjOWZmMDk4MDNmZjMwYTBhYjM2NWYzNDgzYWIxYzIyNWJkNDcyYTJhNzU1NGM5OGFkZjIzJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.VefPn_rzLWOsiVv9holR24rXB_MHrN2CmuXo8Bhpg_4)

找到这个lantern.log文件了，win10 系统，最新的7.8.3版本 ，安装后，还是闪退，敬请帮助解决

**Reply by casperwzy** [reply]
[Uploading lantern.log…]()

**Reply by Letitebe19831018** [reply]
del C:\Users\用户名\AppData\Roaming\Lantern\global.yaml
Start C:\Users\用户名\AppData\Roaming\Lantern\lantern.exe
按照这个脚本我这里恢复了，能够正常打开软件，分享出来，希望能够帮助到大家

**Reply by wangyf9566** [reply]
这个可以用

**Reply by EFLql** [reply]
亲测可用

**Reply by Alinezhao** [reply]
Mac os 有办法解决闪退问题吗？

**Reply by oo7lax1** [reply]
不行啊，只能连接US的，速度慢的不行

**Comment by casperwzy** [comment]
我找到了很早以前使用的蓝灯 3.7.4版本的安装包，试了试没有闪退，但是已经不能使用了
![360截图20240330081326459](https://github.com/getlantern/lantern-forum-cn/assets/25656932/ce396e5f-f3f0-469b-ae46-2e31505af329)

**Comment by casperwzy** [comment]
也就是同一台电脑 安装很早以前的版本没有闪退，我在试试其他版本的蓝灯

**Comment by casperwzy** [comment]
我安装了下载里边的 测试版 没有出现闪退，我现在只能先使用测试版了，还是希望新的版本能解决闪退
![360截图20240330082111968](https://github.com/getlantern/lantern-forum-cn/assets/25656932/073262ff-7562-4d67-9436-fa6d8a0fadbb)
![360截图20240330082133951](https://github.com/getlantern/lantern-forum-cn/assets/25656932/da46d09f-7d68-4e11-904e-7189434c6334)

**Reply by symbolfung** [reply]
你是啥操作系统？

**Reply by mehuba** [reply]
我的MacOS测试版还是闪退

**Comment by L020304** [comment]
测试版以及刚刚下载的7.8.3还是会闪退
操作系统WIN11 杀毒软件是火绒（关闭火绒按照问题仍然存在）

**Reply by symbolfung** [reply]
7.8.3压根就没发布！

**Reply by lishujun** [reply]
最新版本好像是7.8.1都没看见有7.8.3的下载链接，另外怎么查看本地的版本？我查了一下，浏览器和命令行似乎都不提示版本号

**Comment by ycjdixon** [comment]
WIN7版本，下载安装后还是闪退。

**Comment by bing233** [comment]
操作系统WIN11 下载7.8.3版本 安装之后双击基本上没有任何反应

**Reply by bing233** [reply]
另一台使用旧版本蓝灯的win10电脑倒是可以正常使用

**Comment by FL009** [comment]
坏消息，下载了新版本仍旧闪退

**Comment by hataksumo** [comment]
我用的win17系统，依旧闪退。
贴上我的log

My system is windows 11, start up faild too.  The command window show and quickly quit.
my log attached as below

[lantern.log](https://github.com/getlantern/lantern-forum-cn/files/14809622/lantern.log)

**Comment by Luoboan6** [comment]
苹果手机依然不能用，win10依然不能用，有谁可以用了，今天搞了三个小时了，问题没解决，这个网站也是经常登不上，

**Reply by symbolfung** [reply]
是的，我要申请退费了

**Reply by Luoboan6** [reply]
所以我都是一年一年的买，就怕这种出了问题都没人理，之前有一段登不上，软件里发消息都是自动回复，昨天我才发现还可以这里聊，我也在找备胎了，到时候推荐推荐；前几年lantern很稳定的，现在不知道为啥问题频出，尤其我发现有了那个中文网站，不知道是不是代理，好像优惠啥的，在那里买还不能用

**Comment by eryijiu025** [comment]
> https://github.com/getlantern/lantern
> 
> ![image](https://private-user-images.githubusercontent.com/155475142/318117723-4d55f387-520a-491a-a6dd-f2802bc5905d.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTE3NjgyODUsIm5iZiI6MTcxMTc2Nzk4NSwicGF0aCI6Ii8xNTU0NzUxNDIvMzE4MTE3NzIzLTRkNTVmMzg3LTUyMGEtNDkxYS1hNmRkLWYyODAyYmM1OTA1ZC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwMzMwJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDMzMFQwMzA2MjVaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1lNzFkODU3M2QwZTFjZjk3NTNmZjMyMjkxODZkYTUxMmQ0MmIwOGZjMjc5ZTg3NmJhNzYzN2E4NWVkMzZmMzFmJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.AtxYms1R8Z5lwN4Dbvo4kCyGf4TG7UHFXw4w76CDMg8)

你好哪个是7.8.3

**Reply by symbolfung** [reply]
同问

**Comment by LuRenJia3053822459** [comment]
@Mike777Khoo 我看只有7.8.1啊=-=最新的7.8.3在哪里下载？而且我用Downloads中的稳定版和实验班安装包试了也是不行啊

**Reply by symbolfung** [reply]
是的我也一样

**Reply by LuRenJia3053822459** [reply]
Downloads下的所有win安装包我都试过一次了=-=每次卸载我还把安装目录下的所有文件都删除了一遍了不可能出现安装文件存在没覆盖的问题的

**Reply by LuRenJia3053822459** [reply]
@Mike777Khoo 大佬，你们有没有一个预估时间呢？=-=不然一直拖着，大家都不知道啥时候弄好得特地在这里蹲着

**Reply by taoistx** [reply]
@Mike777Khoo  我也只看到7.8.1

**Comment by lisennku** [comment]
win10系统，自动更新后蓝灯打开后，任务栏灰色图标，鼠标移上去就没了

**Reply by symbolfung** [reply]
7.8.3压根就没发布！

**Comment by mehuba** [comment]
FUCK!

**Comment by MaxMKMax** [comment]
各位非常抱歉. 我马上帮你们反馈上去.

**Reply by maioro** [reply]
win10是安装完 kb5035941后出现的问题，急用的可以删掉这个更新试试。

**Reply by MaxMKMax** [reply]
@maioro 谢谢您帮忙解答. 各位可以尝试看他的方法.

**Reply by maioro** [reply]
不好意思，应该是搞错了。
刚又找了台win10电脑测试 升级到7.8.3后可以使用，但装完 5035941和5035845这俩补丁  重启电脑后程序开始闪退，再删掉这2个update补丁重启后还是闪退

**Comment by MaxMKMax** [comment]
各位请等等. 万分抱歉! 我在等美国团队上线. 他们现在是深夜. 谢谢你们的耐心等待!

帮我提交各应用内报告吧. 然后给我你们的 8 位数票号. 也让我知道什么系统? 哪个版本? 谢谢你们

**Reply by Jumpsteelrail** [reply]
辛苦辛苦，也期待问题能尽快解决。

**Reply by MaxMKMax** [reply]
@Jumpsteelrail 不辛苦. 是我们的错. 抱歉啊. 谢谢您的理解和大量! 请等等团队上线.

**Reply by symbolfung** [reply]
无法登入蓝灯，咋提交工单？咋提供8位数票号？

**Reply by casperwzy** [reply]
上午我找的测试版还能用，下午就不行了，又开始闪退了，怎么回事啊

**Comment by nemobyby** [comment]
我要崩溃了，😫从昨晚到现在…

**Reply by nemobyby** [reply]
一开始是win11不行，换了台设备是win10,一开始还行，然后今天早上也崩了

**Comment by ycjdixon** [comment]
根本不是软件的问题，不然不会各个版本同时出现闪退的问题。应该是服务器出现问题了。闪退后没退出浏览器不能访问任何网站了。

**Reply by LuRenJia3053822459** [reply]
不是的=-=我OPPO手机还是可以的

**Reply by LuRenJia3053822459** [reply]
但是我主要都是用电脑的=-=

**Reply by rutby** [reply]
我也是这样， 重装系统之后浏览器好了， 蓝灯还是运行不了

**Comment by UltimateAmitieKaiNiC** [comment]
太生气了！这什么灯呀！关键时刻居然用不了了！

**Reply by mehuba** [reply]
烂灯呀

**Comment by Robertzhuge** [comment]
这2天都这样，闪退

**Comment by symbolyu950920** [comment]
win10我的两台电脑都无法使用，手机端正常使用，需要更新
My two computers can't use lantern software (win10 system). it's good working on my phone.
Need new version！

**Comment by stonesun2023** [comment]
我的三台电脑也都不能用了，尝试安装了各种版本都是闪退。最后找了l-installer-771这个版本 可以临时用免费版本，但是原来的付费版也找不到了！电脑重启之后就又开始各种闪退！！！请尽快解决这个问题。

**Comment by casperwzy** [comment]
> 各位请等等. 万分抱歉! 我在等美国团队上线. 他们现在是深夜. 谢谢你们的耐心等待!
> 
> 帮我提交各应用内报告吧. 然后给我你们的 8 位数票号. 也让我知道什么系统? 哪个版本? 谢谢你们

现在测试版也突然不能用 直接闪退了，上午还能用，这个该怎办呢？

**Comment by blueraincoatli** [comment]
看到这么多人同样情况，我就放心了，折腾了一整天原来不是我自己设备的问题，差点重装系统。。坐等修复吧，我还以为被什么神奇的技术给block掉了

**Comment by woodchuckk** [comment]
因为7.8.2闪退，我上来看看，结果是因为7.8.3推出了啊？现在是怎样，7.8.3版本的稳定吗？win7用了会闪退吗？

**Reply by ppqr258** [reply]
win10系统，安装了7.8.3，还是闪退！

**Comment by YuanGA007** [comment]
win10企业版蓝灯闪退，而且之前的版本需要重新授权专业版，授权专业版成功后地址一直在SG，可SG网速很慢甚至无法使用，请修复闪退问题、授权问题和服务器问题

**Comment by set-aua** [comment]
你好 mike mac os版本还是无法使用  程序无法正常启动

**Comment by MaxMKMax** [comment]
各位好，谢谢您们的反馈。蓝灯已经在正视这个问题了。 我们正在修复问题和进行优化。请您耐心等待。非常抱歉影响了您的体验感。谢谢您的等待与支持。

您可以帮助我们在应用程序中提交报告您的问题吗？您可以访问Lantern应用程序并提交log/ 报告。

1. go to Support (转到支持)
2. select Report an Issue (选择报告问题)
3. choose issue type (选择问题类型)
4. include additional details (包括额外的详细信息)
5. hit Send Report (点击发送报告)

当您在app提交问题报告后您会收到邮件. 邮件标题上有 8 位数号码. 请回来这里留言您的log/ticket号码. 因为我需要拿您的反馈报告号码来和科技团队沟通您遇到的问题. 不然我无法提交您遇到的问题给科技团队.

有了这些资料，我们科技团队便可以尽快解决问题。您可以在留言区给我知道。谢谢。
我们将非常感谢您们!

**Reply by carpediemcc** [reply]
ticket{70932003}
尝试删除了global.yaml文件，暂时不闪退了，但是还是显示连接SG服务器却很慢，什么都打不开，跟以前没有区别。

**Reply by Jumpsteelrail** [reply]
ticket {70933041}
3月29日就出现的问题，蓝灯无法启动，图标呈灰色，鼠标移至图标即崩溃，一直无法修复且重新安装蓝灯后变为免费版，无法授权变回专业版用户。 删除蓝灯文件夹下globel.yaml才能勉强连接上一个免费版的蓝灯发送报告。

**Comment by MaxMKMax** [comment]
有一位用户的建议. 暂时的解决方法. 您们试试看.

我们会尽快修复的. 非常抱歉. 谢谢你们的耐心等待.

如下:

删除，蓝灯安装路径下的global.yaml文件
C:\Users\Administrator.DESKTOP-GINF5UQ\AppData\Roaming\Lantern
这样可以正常打开，但是是免费版本的，你再正常激活，你这次只要不关闭蓝灯就能使用
应该蓝灯的认证出问题了

**Comment by MaxMKMax** [comment]
你们的声音我们都听到了. 请给我们一点时间. 非常抱歉影响了体验感. 也谢谢你们支持蓝灯!

如果各位方便的话. 帮忙提交应用内报告. 然后把 8 位数票号给我们. 这会帮很大的忙.

打个比方:
票号 (Ticket number/ Log number) - 12345678
系统- Window 10, 11/ IOS/ Mac OS/ Android xxx?
版本- 7.8.3

**Reply by Jumpsteelrail** [reply]
票号 70936567
系统 Window 10
版本 7.8.3 (20240305.214140)

**Reply by MaxMKMax** [reply]
@Jumpsteelrail 谢谢您!

**Comment by MaxMKMax** [comment]
你们可以参考一下外面的帖子. 有人分享怎么弄. 也有人说修好了. 
你们更新最新版本了吗?

**Reply by jean-reno** [reply]
应该是好了 我测试了一个下午 没闪退   版本：7.8.1 (20240229.143638)

**Reply by ppqr258** [reply]
7.8.1版本哪儿下载的，我点逛网下载链接，都是下载最新版的！

**Comment by Luoboan6** [comment]
我觉得这次应该是他们服务器的问题， 现在苹果手机版本正常了， 华为手机旧版本也正常了，win的还没好，看不懂他们发的安装教程----- 原始邮件 ***@***.***>发送时间：2024-03-31 ***@***.***>***@***.***>***@***.***>主 题：Re: [getlantern/lantern-forum-cn] 好消息，我们已经修复了 7.8.2 中的崩溃问题，版本 7.8.3 现已推出 https://github.com/getlantern/lantern (Discussion #300)
应该是好了 我测试了一个下午 没闪退 版本：7.8.1 (20240229.143638)

**Comment by MaxMKMax** [comment]
大家好。你们是否仍然面临应用程序崩溃问题或任何其他类似 2 天前的问题？或者已经恢复了？请告诉我。谢谢。

**Reply by chihaohehao** [reply]
我的现在可以正常使用 但是之前重复的信用卡扣费需要退款 麻烦尽快处理 谢谢 我在另一个帖子里@你了 可以看下账单和账号

**Reply by MaxMKMax** [reply]
@chihaohehao 
好的好的. 我明天再帮您通知团队. 如果我忘了你在提醒我. 谢谢您支持蓝灯.

**Reply by Jumpsteelrail** [reply]
票号 70941615
系统 Windows 10
我还有200多天的专业版使用时长，虽然蓝灯不闪退恢复正常了，速度也可以，但是我的专业版账户没有恢复，依然还是免费版，只够油管看俩视频的流量……

**Reply by carpediemcc** [reply]
目前不闪退了，但是连接正常了不超过五分钟，又是跟以往一样，显示连接SG服务器但是什么也打不开。这问题都报了半个多月了，真的不是服务器的问题吗？

**Reply by chihaohehao** [reply]
我再来提醒一下 退款的事情 @Mike777Khoo

**Reply by MaxMKMax** [reply]
@chihaohehao 
您好. 太多帖子了. 我找不到您之前的帖子. 您重新在旧帖子 @ 我一下. 或者重新发邮箱和单据. 谢谢哦!

**Comment by stonesun2023** [comment]
----- 原始邮件 -----
发件人：Mike K ***@***.***>
收件人：getlantern/lantern-forum-cn ***@***.***>
还是没有恢复正常，win下的免费版会显示连通，但是速度极其慢，经常没有流量，根本无法正常上网使用。
请尽快出一个稳定的更新版本，现有的版本管理太乱了。

good luck！

抄送人：stonesun2023 ***@***.***>,  Comment ***@***.***>
主题：Re:_[getlantern/lantern-forum-cn]_好消息，我们已经修复了_7.8.2_中的崩溃问题，版本_7.8.3_现已推出_https://github.com/getlantern/lantern_(Discussion_#300)
日期：2024年03月31日 22点27分


大家好。你们是否仍然面临应用程序崩溃问题或任何其他类似 2 天前的问题？或者已经恢复了？请告诉我。谢谢。

**Comment by Luoboan6** [comment]
我用的beta版，目前恢复正常了----- 原始邮件 ***@***.***>发送时间：2024-03-31 ***@***.***>***@***.***>***@***.***>主 题：Re: [getlantern/lantern-forum-cn] 好消息，我们已经修复了 7.8.2 中的崩溃问题，版本 7.8.3 现已推出 https://github.com/getlantern/lantern (Discussion #300)
7.8.1版本哪儿下载的，我点逛网下载链接，都是下载最新版的！

**Comment by yanggaojun101** [comment]
我装了7.8.3，不闪退，但是变成了免费版，我的邮箱收不到验证码，怎么解决啊？yanggaojun101@126.com

**Reply by MaxMKMax** [reply]
@yanggaojun101 您好. 我帮您问看. 请您明天检查邮箱.

**Reply by yanggaojun101** [reply]
mike，进展怎么样了

**Reply by MaxMKMax** [reply]
![image](https://github.com/getlantern/lantern-forum-cn/assets/155475142/1f366654-8c6e-4600-a2f7-6dcfc7ea740c)

@yanggaojun101 支付部门说您不是专业版用户呢. 系统里您没有购买也. 可能之前没付款成功. 还是您能提供我付款单据吗?

**Reply by Jumpsteelrail** [reply]
@Mike777Khoo 您能帮我解决一下我的专业版不能使用的问题吗？已经两天了……没出问题之前显示的时200多天而且我在之前刚授权了一台电脑。

**Reply by Jumpsteelrail** [reply]
@Mike777Khoo 您能帮我解决一下我的专业版不能使用的问题吗？已经两天了……没出问题之前显示的时200多天而且我在之前刚授权了一台电脑。

**Comment by Luoboan6** [comment]
我是win10系统， 下载的Beta版，不知道啥版本，用手机授权新设备可以了，现在正常----- 原始邮件 ***@***.***>发送时间：2024-04-01 ***@***.***>***@***.***>***@***.***>主 题：Re: [getlantern/lantern-forum-cn] 好消息，我们已经修复了 7.8.2 中的崩溃问题，版本 7.8.3 现已推出 https://github.com/getlantern/lantern (Discussion #300)
4月1日早9点，安装前也把文件也清除了，然后安装了7.83，但还是闪退。

**Comment by wittmann81** [comment]
现在进入已经可以了，不会闪退了。谢谢！

**Reply by MaxMKMax** [reply]
@wittmann81 
嗯嗯. 那就好. 不客气. 谢谢您支持蓝灯!

**Comment by stonesun2023** [comment]
请问你装的哪个版本？是在win下面使用么？
----- 原始邮件 -----
发件人：wittmann81 ***@***.***>
收件人：getlantern/lantern-forum-cn ***@***.***>
抄送人：stonesun2023 ***@***.***>,  Comment ***@***.***>
主题：Re:_[getlantern/lantern-forum-cn]_好消息，我们已经修复了_7.8.2_中的崩溃问题，版本_7.8.3_现已推出_https://github.com/getlantern/lantern_(Discussion_#300)
日期：2024年04月01日 12点35分


现在进入已经可以了，不会闪退了。谢谢！

**Reply by wittmann81** [reply]
看版本是7.83，Win10运行。今天上午才正常。

**Comment by PrinceChen** [comment]
macOS Sonama 14.4.1 
lantern 7.8.3 或 7.8.1 卸载重装，重启电脑，都会闪退。上周五开始的，持续了好几天了

**Reply by MaxMKMax** [reply]
@PrinceChen 好的. 今晚团队上线了. 我会再帮你们反馈. 如果方便的话再帮我提交应用内报告. 然后给我 8 数票号. 谢谢您!

**Reply by PrinceChen** [reply]
@Mike777Khoo 反馈一下，我刚刚下载了7.8.4的版本，可以正常使用了。
安装完，打开后，一开始显示的免费版，然后选择用邮箱授权，自动检测到本设备属于专业版，确认一下直接就切到专业版了

**Reply by txwgoogol** [reply]
7.8.4哪里下载的

**Reply by PrinceChen** [reply]
@txwgoogol 就平常的下载链接，顶部那个点进去就可以下载到。版本日期是昨天的7.8.4 (20240401.092633)

**Reply by Evan7411** [reply]
macOS系统下第二个链接是7.8.4，第一个链接是7.8.3.   7.8.4暂可以用！

**Comment by antheeee** [comment]
请问7.8.3在哪？你给出的链接页面里只有7.8.1，或者说如何操作才能升级成7.8.3？
而且变成了免费版，如何才能恢复成专业版？

**Comment by fuhuasheng1** [comment]
我的MacOS 蓝灯现在一登录就崩溃掉，换成老版本，不崩溃了，但连不上蓝灯了

**Comment by wraywell** [comment]
试了最新的7.8.3还是无法使用，app无法打开
Win11的系统
这个问题也是出来这么多天了，要么搞研发的太弱或者不当回事，要么就是这个问题比较麻烦...

**Comment by stonesun2023** [comment]
7.8.4版本出来了？
麻烦请帮忙提供下7.8.4的版本的下载链接，谢谢！


--------------------------------

----- 原始邮件 -----
发件人：PrinceChen ***@***.***>
收件人：getlantern/lantern-forum-cn ***@***.***>
抄送人：stonesun2023 ***@***.***>,  Comment ***@***.***>
主题：Re:_[getlantern/lantern-forum-cn]_好消息，我们已经修复了_7.8.2_中的崩溃问题，版本_7.8.3_现已推出_https://github.com/getlantern/lantern___(Logs_also_here)_(Discussion_#300)
日期：2024年04月02日 15点48分


@Mike777Khoo 反馈一下，我刚刚下载了7.8.4的版本，可以正常使用了。

安装完，打开后，一开始显示的免费版，然后选择用邮箱授权，自动检测到本设备属于专业版，确认一下直接就切到专业版了

**Comment by carpediemcc** [comment]
好消息：暂时不闪退了；坏消息：还是很慢。这个SG服务器感觉跟假的一样，半个多月了一直是这样，每次跟开盲盒一样，都不知道哪一次能不卡，能不能抓紧修一修啊@Mike777Khoo

**Comment by shantutu1026** [comment]
@Mike777Khoo  你还在继续跟进此事么？到底要如何解决有没有一个统一的说法？我使用macOS系统，专业版2年付费用户。蓝灯版本7.8.3闪崩问题已经没有，但无法登录专业版。我不知道哪里能找到激活码，并且因为清理旧邮件也找不到票号。只有付费记录的截图。请提供解决方案。出来冒个泡吧@Mike777Khoo 。

**Reply by MaxMKMax** [reply]
@shantutu1026 不好意思, 我刚上线. 您方便提供我您有账号邮箱吗? 我让付款团队给您弄个新的激活码.

**Reply by shantutu1026** [reply]
多谢@Mike777Khoo，作为老用户我忘了当时用哪个邮箱注册的。请分别给shantutu1026@gmail.com  和  38047034@qq.com 发送吧。顺便请告知激活码用法。谢谢！

**Reply by shantutu1026** [reply]
@Mike777Khoo ，我还没有收到激活码呀。是不是团队太忙忘了？目前免费版但似乎不限流了，但没法授权别的设备呀。帮忙解决一下呗。

**Comment by stonesun2023** [comment]
确认是win的版本么？最上面的安装包时间戳是3月6号的。

 



 方便的话，发个下载链接。谢谢先！





 

 --------------------------------
 

 
 
  
 

 

 

 
 
  
 
  ----- 原始邮件 -----
  
***@***.***>
  
***@***.***>等3人
  
主题：Re:_[getlantern/lantern-forum-cn]_好消息，我们已经修复了_7.8.2_中的崩溃问题，版本_7.8.3_现已推出_https://github.com/getlantern/lantern___(Logs_also_here)_(Discussion_#300)
  
日期：2024年04月02日 17:58:28
  

 
 

  
 @txwgoogol 就平常的下载链接，顶部那个点进去就可以下载到。版本日期是昨天的7.8.4 (20240401.092633)

**Comment by Jumpsteelrail** [comment]
@Mike777Khoo 您能帮我解决一下我的专业版不能使用的问题吗？已经两天了……没出问题之前显示的时200多天而且我在之前刚授权了一台电脑。
邮箱是 jumpsteelrail@126.com
